files/initramfs-tools/union-mount.j2

```diff
@@ -39,10 +39,38 @@ set_tmpfs_log_partition_size()
   [ $maxsize -le $varlogsize ] && varlogsize=$maxsize
 }
 
+whitelist_rw_folder()
+{
+  image_dir=$1
```

Comment:
If it doesn't bother you too much. can you switch to 4 space indentation? Most of our files are 4 space indentations.

Reply:
Fixed.